pkgs/by-name/pl/plumber/package.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "plumber";
+  version = "2.5.2";
+
+  src = fetchFromGitHub {
+    owner = "streamdal";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-ftXLipJQjRdOSNO56rIRfAKKU0kHtAK85hgcT3nYOKA=";
+  };
+
+  vendorHash = null;
+  doCheck = false; # connection tests create a config file in user home directory which fails in the sandbox
```

Comment:
DIdn't see a way to skip just a directory of tests.. not sure how frowned upon this is.

Reply:
can try something like:
```nix
  preCheck = ''
    export HOME="$(mktemp -d)"
  '';
```

Reply:
It worked. Thanks @a-n-n-a-l-e-e!